launch_testing_ros/test/examples/repeater.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import String
+
+
+class Repeater(Node):
+
+    def __init__(self):
+        super().__init__('repeater')
+        self.count = 0
+        self.subscription = self.create_subscription(
+            String, 'input', self.callback, 10
+        )
+        self.publisher = self.create_publisher(String, 'output', 10)
+
+    def callback(self, input_msg):
+        self.get_logger().info('I heard: [%s]' % input_msg.data)
+        output_msg_data = input_msg.data
+        self.get_logger().info('Publishing: "{0}"'.format(output_msg_data))
```

Comment:
f-strings for new code:

```suggestion
        self.get_logger().info(f'I heard: [{input_msg.data}]')
        output_msg_data = input_msg.data
        self.get_logger().info(f'Publishing: "{output_msg_data}"')
```